test/__snapshots__/invalid-options.test.js.snap

```diff
@@ -3,9 +3,9 @@
 exports[`errors 1`] = `
 Array [
   "Error: main.0c220ec66316af2c1b24.js from UglifyJs
-DefaultsError: \`invalid-option\` is not a supported option",
+\`invalid-option\` is not a supported option",
```

Comment:
can you try changing `(this line)[https://github.com/webpack-contrib/uglifyjs-webpack-plugin/blob/master/test/helpers.js#L68] to: 

```js
const str = exports.removeCWD(error.message)
```